tools/ci/run_tc.py

```diff
@@ -146,25 +146,20 @@ def start_dbus():
     # https://source.chromium.org/chromium/chromium/src/+/main:content/app/content_main.cc;l=220;drc=0bcc023b8cdbc073aa5c48db373810db3f765c87.
     os.environ["DBUS_SESSION_BUS_ADDRESS"] = "autolaunch:"
 
-
 def install_chrome(channel):
-    if channel in ("experimental", "dev"):
-        deb_archive = "google-chrome-unstable_current_amd64.deb"
-    elif channel == "beta":
-        deb_archive = "google-chrome-beta_current_amd64.deb"
-    elif channel == "stable":
-        deb_archive = "google-chrome-stable_current_amd64.deb"
-    else:
-        raise ValueError("Unrecognized release channel: %s" % channel)
-
-    dest = os.path.join("/tmp", deb_archive)
-    deb_url = "https://dl.google.com/linux/direct/%s" % deb_archive
-    with open(dest, "wb") as f:
-        get_download_to_descriptor(f, deb_url)
-
-    run(["sudo", "apt-get", "-qqy", "update"])
-    run(["sudo", "gdebi", "-qn", "/tmp/%s" % deb_archive])
-
+    # Interpret experimental channels as Canary.
+    if channel in ("experimental", "nightly", "canary"):
```

Comment:
What happens for the stable and beta cases now?

Reply:
This is just a confirmation that we're interpreting "experimental" and "nightly" as "canary", since that is where the files will be when installing. Beta and stable should still be handled normally. 🙂

Reply:
I'm confused in the same way @jcscottiii is. It looks like this removes all support for installing non-Canary versions of Chrome. But are those not still required for stable/beta runs in CI?

Also, for installation that doesn't involve a debian package we shouldn't have any code in `run_tc.py`, but should instead pass `--install-browser` to `wpt run` (look at what we're doing for Firefox). Ideally this environment setup wrapper wouldn't need to know anything about which browser we're going to end up running.

Reply:
Ah, I'm sorry to both of you 😓 - I missed that the `channel` variable is not being used in the `./wpt install` statement, and canary is hardcoded in. I thought the confusion was with the channel interpretation logic. This is now fixed.

As for Firefox, I believe it is not installed using `wpt run` for the integration tests, which also had me scratching my head a bit looking to mimic the implementation - but it turns out it the installation is [specified in the Dockerfile via apt-get](https://github.com/web-platform-tests/wpt/blob/master/tools/docker/Dockerfile#L51-L56)

Reply:
> As for Firefox, I believe it is not installed using wpt run for the integration tests

Huh, if so that seems very much like a bug / regression. The only reason we install Firefox in the Docker image is to ensure we have all the right dependencies. Maybe we should remove the Firefox binary to stop that happening.